src/components/Navbar.tsx

```diff
@@ -8,17 +14,13 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink to="/" className={selectedPage}>
             Home
-          </a>
+          </NavLink>
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
+          <NavLink aria-current="page" className={selectedPage} to="/people">
```

Comment:
The 'aria-current="page"' attribute should only be used on the currently active page link. Consider moving this attribute to the 'NavLink' that corresponds to the current page.